unitOneAssessmentPractice.js

```diff
@@ -3,39 +3,100 @@ let assert = require('assert')
 // Question One:
 
 // Write a function called double that doubles a number
-
-// Uncomment out the next line to test your solution
+function double(num) {
+  return num*2;
+}
+double(14);
+// // Uncomment out the next line to test your solution
 // runQ1Tests()
 
 // Question Two:
 
 // Write a function called containsSeven that returns whether or not a number has an sevens in it.
+function containsSeven(num) {
+ let newNum = num.toString();
+ for (let i = 0; i < newNum.length; i++) {
+   if (newNum[i] === "7") {
+    return true;
+   } else {
+    i++;
+   }
+ }
+}
+ containsSeven(5378);
 
 // Uncomment out the next line to test your solution
 // runQ2Tests()
 
 // Question Three:
 
 // Write a function called capitalizeTheAs the capitalizes all of the lowercase "a"s in a string leaving all other characters the same
+function capitalizeTheAs(str) {
+  for (let i = 0; i < str.length; i++) {
+    if (str[i] === "a") {
+    return str.replace("a", "A");
+    }
+  }
+}
+capitalizeTheAs("giraffe");
 
 // Uncomment out the next line to test your solution
 // runQ3Tests()
 
 // Question Four:
 // Write a function called largest that returns the largest value in an array
+function largest(arr) {
+  let newLargest = "";
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i] > newLargest) {
+      newLargest = arr[i];
+    } else {
+      i++;
+    }
+  }
+  return newLargest;
+}
+
+largest([8,1,12,14,6]);
 
 // Uncomment out the next line to test your solution
 // runQ4Tests()
 
 // Question Five:
 // Write a function called average that returns the average of an array
+function average(arr) {
+  let sum = 0;
+  for (let i=0; i < arr.length; i++) {
+    sum += arr[i];
+  }
+  return (sum/arr.length);
+}
 
+average([2,4,6,8]);
 // Uncomment out the next line to test your solution
 // runQ5Tests()
 
 // Question Six:
 // Write a function called mode that returns the most frequently occurring number in an array
 // HINT: Use an object where the keys are the numbers, and the values are how many times they appear in the array.
+// function mode(arr) {
```

Comment:
You got the right idea of using two loops for this problem. One loop is to create an object with all the numbers counted in an object (ex: {"3": 4, "4": 1, "6": 2, "5": 2}. 
The second loops is to find the highest value which represented most frequently contained number.